JDI.Light/JDI.Light.Tests/Tests/Common/FileInputTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using JDI.Light.Tests.UIObjects;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using static JDI.Light.Jdi;
+
+namespace JDI.Light.Tests.Tests.Common
+{
+    [TestFixture]
+    class FileInputTests : TestBase
+    {
+        //Tests require administrator access
+
+        [SetUp]
+        public void SetUp()
+        {
+            Logger.Info("Navigating to HTML5 page.");
+            TestSite.Html5Page.Open();
+            TestSite.Html5Page.CheckTitle();
+            Logger.Info("Setup method finished");
+            Logger.Info("Start test: " + TestContext.CurrentContext.Test.Name);
+        }
+
+        private string CreateFile(string filename)
+        {
+            string filepath = Path.Combine(Directory.GetCurrentDirectory(), filename);
+            var sw = File.CreateText(filepath);
+            sw.WriteLine("hello world");
+            sw.Close();
+            return filepath;
+        }
+
+        [Test]
+        public void FileInputTest()
+        {
+            string filename = "test.txt";
+            TestSite.Html5Page.FileInput.SelectFile(CreateFile(filename));
+            var uploadedFile = TestSite.Html5Page.FileInput.GetAttribute("value");
+            Jdi.Assert.Contains(uploadedFile, filename);
+        }
+
+        [Test]
+        public void DownloadTest()
+        {
+            string userRoot = System.Environment.GetEnvironmentVariable("USERPROFILE");
```

Comment:
manespace is not required here:
![image](https://user-images.githubusercontent.com/6689884/54680652-74f62780-4b1b-11e9-8ef9-a7256c9a7f92.png)